gladosTTS.py

```diff
@@ -105,8 +104,7 @@ def speak(line, cache=False):
 				eye_position_random()
 
 	    	# Speak
-			ps('output.wav')
-
+                        call(["aplay", "./output.wav"])

```

Comment:
Please fix indentation to use tabs instead of spaces on row 107. My Python freaks out if these are mixed.
Otherwise script is good and worked.

Reply:
My python freaked out also before I changed them to tabs (this is what actually took most of the time for me to find out :D ). I'll check if I've messed something there.

Reply:
I edited the file but I'm not 100% sure I did it right (still learning github)...